src/NLog/Internal/FileInfoExt.cs

```diff
@@ -43,19 +43,16 @@ public static DateTime GetLastWriteTimeUtc(this FileInfo fileInfo)
 #if !SILVERLIGHT
             return fileInfo.LastWriteTimeUtc;
 #else
-            return fileInfo.LastWriteTime;
+            return fileInfo.LastWriteTime.ToUniversalTime();
```

Comment:
Well it only happens for silverlight-applications, and since the method should return Utc, then it would make sense to do it correct. But I will remove it, since it is not important to me.